tests/console/vim.pm

```diff
@@ -17,6 +17,10 @@ use testapi;
 use version_utils 'is_jeos';
 
 sub run {
+    if (check_var('DISTRI', 'opensuse') and check_var('VERSION', '15.0') and (check_var('ARCH', 'aarch64') or check_var('ARCH', 'ppc64le'))) {
```

Comment:
Fyi I fixed the missing vim issue last night, today's build should have it (it's on the media)